anomalib/models/dfm/model.py

```diff
@@ -88,10 +84,89 @@ def validation_step(self, batch, _):  # pylint: disable=arguments-differ
         Returns:
           Dictionary containing FRE anomaly scores and ground-truth.
         """
+        batch["pred_scores"] = self.model(batch["image"])
+
+        return batch
+
+
+@MODEL_REGISTRY
+class Dfm(AnomalibModule):
+    """DFM: Deep Featured Kernel Density Estimation.
```

Comment:
This title mentions DFKDE instead of DFM. Same for the other docstrings in this file